apps/frontend/src/features/allowlists/presentation/components/allowlist.tsx

```diff
@@ -1,12 +1,31 @@
+import { CHAIN_DETAILS } from '../../../../core/utilities/Constants';
 import axios from 'axios';
 import React, { useState } from 'react';
+import ProjectUtils from '../../../../core/utilities/ProjectUtils';
 
 const Allowlist = (props) => {
   const end_date = props.end_date.substring(0, 10);
   const [editMode, setEditMode] = useState(false);
   const [description, setDescription] = useState(props.description || '');
   const [endDate, setEndDate] = useState(end_date);
 
+  const signUp = async () => {
+    const url = `/api/v1/allowlist/join/${props.id}`;
+
+    const data = await ProjectUtils.signMessage(
+      CHAIN_DETAILS.RPC_ADDRESS[props.walletStore.selectedNetwork],
+      props.walletStore.ledger.offlineSigner,
+      props.walletStore.getAddress(),
+    );
+
+    try {
+      const res = await axios.post(url, data);
+      alert('success');
```

Comment:
is this for testing purposes only?

Reply:
yes, this alert is for testing purposes only